ballerina-tests/tests/76_number_types_pb.bal

```diff
@@ -15,10 +15,11 @@
 // under the License.
 
 import ballerina/grpc;
+import ballerina/lang.'int;
```

Comment:
btw, why do we need this import?
We can directly do `int:Unsigned32` right?

Reply:
Removed in [4d4df2d](https://github.com/ballerina-platform/module-ballerina-grpc/pull/1351/commits/4d4df2db6c2ddf1ecdf4f9f770273ad7dc2e0961)